packages/edit-site/src/store/reducer.js

```diff
@@ -142,19 +142,16 @@ export function navigationPanel(
 		case 'SET_IS_NAVIGATION_PANEL_OPENED':
 			return {
 				...state,
-				menu: ! action.isOpen ? MENU_ROOT : state.menu, // Set menu to root when closing panel.
 				isOpen: action.isOpen,
 			};
 		case 'SET_IS_LIST_VIEW_OPENED':
 			return {
 				...state,
-				menu: state.isOpen && action.isOpen ? MENU_ROOT : state.menu, // Set menu to root when closing panel.
 				isOpen: action.isOpen ? false : state.isOpen,
 			};
 		case 'SET_IS_INSERTER_OPENED':
 			return {
 				...state,
-				menu: state.isOpen && action.value ? MENU_ROOT : state.menu, // Set menu to root when closing panel.
```

Comment:
Seemed unnecessary after #30098, but perhaps this is going on too much of a tangent for this PR as it didn't seem to be interfering with `openNavigationPanelToMenu`. Looks like it'd require test updates.